tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -586,6 +588,14 @@ struct LaunchBatchMatMul<GPUDevice, Scalar> {
                                     std::is_same_v<Scalar, Eigen::bfloat16>;
     using Coefficient = std::conditional_t<is_16bit_input, float, Scalar>;
 
+    se::blas::CallContext call_context =
+      se::blas::CallContext::kNone;
+    if (grad_x) {
```

Comment:
Could they ever both be true?

Reply:
@cantonios  As I understand both of them can't be true at the same time.

Reply:
Can you add a check for that then (i.e. an `OP_REQUIRES`)